### PR TITLE
Feature/store

### PR DESCRIPTION
**Pull request by tgkd**
redux store
react-router
axios http req

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -68,58 +68,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
-
```
Comment (victorsmirnov) [inline pom.xml]: @tgkd please revert the changes in this file. We did it locally only to skip JS compilation in project build when JS failed to compile. We need this to build production JAR.

File: src/main/js/app/api/config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+export default {
+    baseUrl: 'http://localhost:8080/api/',
```
Comment (victorsmirnov) [inline src/main/js/app/api/config.js]: @tgkd I think base URL should be `/api/` because API is on the same host.

**Comment by tgkd** [comment]
@victorsmirnov About strict mode. According to [this](http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code) specification ES6 modules are always in strict mode.
